app/globals.css

```
@@ -5,75 +5,20 @@
 @custom-variant dark (&:where(.dark, .dark *));
 
 @theme {
```

Stylelint (17.6.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)